diff --git a/lua/java/startup/mason-dep.lua b/lua/java/startup/mason-dep.lua
index f05e3a5..e65fcc3 100644
--- a/lua/java/startup/mason-dep.lua
+++ b/lua/java/startup/mason-dep.lua
@@ -31,14 +31,14 @@ function M.install(config)
 end
 
 function M.refresh_and_install(packages)
-	vim.schedule(function()
-		-- lazy covers mason
-		-- https://github.com/nvim-java/nvim-java/issues/51
-		lazy.close_lazy_if_opened()
-
-		mason_ui.open()
-		notify.warn('Please close and re-open after dependecies are installed')
-	end)
+	-- vim.schedule(function()
+	-- 	-- lazy covers mason
+	-- 	-- https://github.com/nvim-java/nvim-java/issues/51
+	-- 	lazy.close_lazy_if_opened()
+	--
+	-- 	mason_ui.open()
+	-- 	notify.warn('Please close and re-open after dependecies are installed')
+	-- end)
 
 	mason_util.refresh_registry()
 	mason_util.install_pkgs(packages)
